deny.toml

```diff
@@ -56,4 +53,4 @@ unknown-registry = "deny"
 unknown-git = "deny"
 
 [sources.allow-org]
-github = ["stackabletech", "teozkr"]
\ No newline at end of file
+github = ["stackabletech", "teozkr"]
```

Comment:
Do we get anything from your repo here still?

Reply:
The patched unix domain stream-friendly h2 build.